PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "toxic-birthday",
+   "metadata": {},
+   "source": [
+    "# Extracting and Plotting Position-Velocity Diagrams\n",
+    "\n",
+    "## Authors\n",
+    "Adam Ginsburg\n",
+    "\n",
+    "## Learning Goals\n",
+    "* Extract a position-velocity diagram from a cube using both pixel and sky coordinates\n",
+    "* Display the position-velocity diagram with appropriately labeled coordinates\n",
+    "* Display the extraction path on the plots\n",
+    "\n",
+    "## Keywords\n",
+    "cube, pv-diagram\n",
+    "\n",
+    "## Summary\n",
+    "In this tutorial, we will extract position-velocity (PV) diagrams from a cube and plot them."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "apart-slope",
+   "metadata": {},
+   "source": [
+    "### Header material\n",
+    "\n",
+    "We import tools from several packages up front:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "living-divorce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pylab as pl\n",
+    "import numpy as np\n",
+    "from astropy.visualization import quantity_support\n",
+    "from astropy import units as u\n",
+    "from astropy import wcs\n",
+    "\n",
+    "# set so that these display properly on black backgrounds\n",
+    "pl.rcParams['figure.facecolor']='w'\n",
+    "\n",
+    "from spectral_cube import SpectralCube\n",
+    "\n",
+    "from pvextractor import extract_pv_slice, Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "minor-amateur",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# retrieve and open a cube from astropy-data\n",
+    "cube = SpectralCube.read('http://www.astropy.org/astropy-data/l1448/l1448_13co.fits')\n",
+    "cube"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "polar-baseball",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# show an image in pixel coordinates\n",
```

Comment:
I think this needs its own cell, too, with some added explanation:
- Show single channel to find where to draw the path
- Show in pix units so it's easier to define the path from the pix coords in the mpl viewer